File: programme/random-number/random-number.js
// Write a programme to generate a random number in range n1, n2

const n1 = 1;
const n2 = 100;

function getRandomNumberBetween(n1, n2) {
  return Math.floor(Math.random() * (n2 - n1 + 1) + n1);
}

console.log("Start range   :", n1);
console.log("End   range   :", n2);
console.log("Random number :", getRandomNumberBetween(n1, n2));

File: programme/random-number/random-number.jl
# Write a programme to generate a random number in range n1-n2
function output_in_range()
  print("Enter start range : ")
  start_range = readline()

  print("Enter end   range : ")
  end_range = readline()

  println("\nOutput : ", rand((parse(Int64, start_range)):parse(Int64, end_range))) #We use parsing in order to turn a string input into an integer because rand works only with ints
end

output_in_range()
